composer.json

```diff
@@ -15,6 +15,27 @@
 		"guzzlehttp/promises": "~1.0",
 		"psr/log": "~1.0"
 	},
+	"require-dev": {
+		"phpunit/phpunit": "~4.8.0|~5.3.0",
+		"jakub-onderka/php-parallel-lint": "0.9.2"
+	},
+	"suggest": {
+		"etsy/phan": "Allows running static analysis on the package (requires PHP 7+)"
```

Comment:
So, I was trying to figure out how to do this.
I was phan running on CI, but I can't add it to the composer.json as it specifies php 7.0+, so this package would then not be installable with php 5.5 etc..
I put it in suggest as this repo does indeed have a phan config and you could use it..
for CI I guess I will add a step that requires phan in cases where PHP is greater than 7?
Or maybe make a separate job just for phan and have a larger build matrix..